### PR TITLE
Suggesting - Batch 1.

### PR DESCRIPTION
**Pull request by chicken-biryani**
This pull request is a few suggestions for the introduction section of the FLINT handbook.

The introduction written by Amarachi is SO GOOD. I did not feel lost as a reader.

I am adding a few comments via review to add a few more suggestions to make a reader feel more comfortable/to explain why the suggestion.

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -8,27 +8,24 @@ FLINT Handbook
...
-
-With climate change resulting in hotter temperatures, more severe storms, increased drought, rising sea levels, higher health risks, etc., more attention has been given to mitigating the effects of human-induced climate change, cue FLINT.
+Greenhouse gases that wrap the globe are released as a result of land usage related to agriculture, forestry, and human growth. More focus has been placed on reducing the effects of human-induced climate change, which includes hotter temperatures, more intense storms, increasing drought, rising sea levels, and increased health hazards. This brings us to FLINT.
 
 The Full Lands Integration Tool (FLINT) is a moja global software used for measuring, reporting, and verifying (MRV) greenhouse gas emissions and removals from forestry, agriculture, and other land uses (AFOLU).
 
 Consistent with the guidelines by the Intergovernmental Panel on Climate Change (IPCC), FLINT utilises multiple data streams to forecast future greenhouse gas emissions and removals.
```
Comment (chicken-biryani) [inline index.rst:20]: I think we can add a section that talks about "What is the FLINT handbook? Why shall one read it/ who is the handbook meant for?" Either here or at the beginning.

File: index.rst [hunk 2]
```diff
@@ -8,27 +8,24 @@ FLINT Handbook
...
 
-Land use associated with agriculture, forestry, and human development results in the emission of greenhouse gases that envelop the earth. Fossil fuels are a major source of emissions, but they are not the focus of FLINT.
-
-With climate change resulting in hotter temperatures, more severe storms, increased drought, rising sea levels, higher health risks, etc., more attention has been given to mitigating the effects of human-induced climate change, cue FLINT.
+Greenhouse gases that wrap the globe are released as a result of land usage related to agriculture, forestry, and human growth. More focus has been placed on reducing the effects of human-induced climate change, which includes hotter temperatures, more intense storms, increasing drought, rising sea levels, and increased health hazards. This brings us to FLINT.
```
Comment (chicken-biryani) [inline index.rst:16]: I have removed any reference to 'FLINT' above this paragraph. The word FLINT can get intimidating and I thought that we should not talk about it till we don't introduce it.

File: index.rst [hunk 3]
```diff
@@ -8,27 +8,24 @@ FLINT Handbook
...
 
 The Full Lands Integration Tool (FLINT) is a moja global software used for measuring, reporting, and verifying (MRV) greenhouse gas emissions and removals from forestry, agriculture, and other land uses (AFOLU).
 
 Consistent with the guidelines by the Intergovernmental Panel on Climate Change (IPCC), FLINT utilises multiple data streams to forecast future greenhouse gas emissions and removals.
 
 
-**What this handbook will contain.**
-
-The FLINT handbook will discuss understanding climate science, the problems climate change poses to our environment, and steps we can take to mitigate the effects.
+**What is included in this handbook.**
```
Comment (chicken-biryani) [inline index.rst:22]: I made the tense to present tense rather than future tense.

File: index.rst [hunk 4]
```diff
@@ -8,27 +8,24 @@ FLINT Handbook
...
+We are trained to expect or welcome change, but what should we do if those changes result in unfavourable or unfavorable circumstances? Climate change is a prime illustration of such a change.
 
-Climate change can be referred to as long-term temperature and weather pattern shifts. Although these shifts may be natural, human activities like burning fossil fuels and deforestation have been the major cause of climate change since the 1800s. Other causes of climate change include solar variability, volcanic activity, etc.
+Long-term changes in temperature and weather patterns are referred to as climate change. Although these changes may be natural, since the 1800s, human actions like the combustion of fossil fuels and deforestation have been the primary contributors to climate change. Solar variation, volcanic activity, and other factors are some other natural sources of climate change.
 
-Land use associated with agriculture, forestry, and human development results in the emission of greenhouse gases that envelop the earth. Fossil fuels are a major source of emissions, but they are not the focus of FLINT.
```
Comment (chicken-biryani) [inline index.rst:15]: Combined the paragraphs so that the flow doesn't break.

**Comment by chicken-biryani** [comment]
@Iheanacho-ai amazing work! What you wrote in the introduction section is engaging.

I would like to hear your and @aornugent 's thought on the 'What is the Flint handbook?' part.

File: index.rst (hunk 4 above)
Comment (Iheanacho-ai) [reply index.rst:15]: > This pull request is a few suggestions for the introduction section of the FLINT handbook.
> 
> The introduction written by Amarachi is SO GOOD. I did not feel lost as a reader.
> 
> I am adding a few comments via review to add a few more suggestions to make a reader feel more comfortable/to explain why the suggestion.

thank you so much!

File: index.rst (hunk 2 above)
Comment (Iheanacho-ai) [reply index.rst:16]: Alright, thank you!

File: index.rst (hunk 1 above)
Comment (Iheanacho-ai) [reply index.rst:20]: Alright, thank you!

File: index.rst (hunk 3 above)
Comment (Iheanacho-ai) [reply index.rst:22]: Alright, thank you!

**Comment by chicken-biryani** [comment]
I'll stack up the things to discuss in an issue, so that it's not lost.

File: index.rst (hunk 1 above)
Comment (Iheanacho-ai) [reply index.rst:20]: hey but is this not the same as the section, what is included in this handbook

**Pull request closed without merging**